vagrant-pxe-harvester/settings.yml

```diff
@@ -60,28 +60,28 @@ harvester_network_config:
       mac: 02:00:00:0D:62:E2
       cpu: 8
       memory: 16384
-      disk_size: 250G
+      disk_size: 500G
       vagrant_interface: ens5
       mgmt_interface: ens6
     - ip: 192.168.0.31
       mac: 02:00:00:35:86:92
       cpu: 8
       memory: 8192
-      disk_size: 250G
+      disk_size: 500G
       vagrant_interface: ens5
       mgmt_interface: ens6
     - ip: 192.168.0.32
       mac: 02:00:00:2F:F2:2A
       cpu: 8
       memory: 16384
-      disk_size: 250G
+      disk_size: 500G
       vagrant_interface: ens5
       mgmt_interface: ens6
     - ip: 192.168.0.33
       mac: 02:00:00:A7:E6:FF
       cpu: 8
       memory: 8192
-      disk_size: 250G
+      disk_size: 500G
```

Comment:
This looks good :smile: as this seems to prevent some issues that arise during upgrade at times.
Could we additionally implement this change on the `settings.yml` in the airgap folder?
https://github.com/harvester/ipxe-examples/blob/8a6a36aeefb01f4f2ff30a30fb0a30d50761ce62/vagrant-pxe-airgap-harvester/settings.yml#L60-L88

Reply:
Thanks @irishgordo! Fixed!